tests/test_flat.py

```diff
@@ -19,6 +19,7 @@ def get_random_patient(patient_id: int, include_metadata=True) -> meds.Patient:
     epoch = datetime.datetime(1990, 1, 1)
     birth = epoch + datetime.timedelta(days=random.randint(100, 1000))
     current_date = birth
+    random.seed(patient_id)
```

Comment:
seed needs to be higher to achieve its goal? `random.randint(100, 1000))` is called above.